toxcore/MDevice.c

```diff
@@ -570,31 +570,37 @@ static int actually_send_friend_list(Tox *tox, uint32_t dev_num)
         }
     }
 
-    int i;
+    int i, j;
     uint8_t packet[sizeof(uint8_t) * 2 + sizeof(uint8_t) * crypto_box_PUBLICKEYBYTES];
 
     for (i = 0; i < tox->m->numfriends; ++i) {
-        if (!tox->m->friendlist[i].status) {
-            /* Currently we sync all friends who we've send a friend request to.
-               Q: do we want to be "< FRIEND_CONFIRMED?"  So we only send to
-                  confirmed friends?
-               We don't (yet) sync the nospam, or the friend request message */
-            continue;
-        }
+        for(j = 0; j < tox->m->friendlist[i].dev_count; j++){
+            if (!tox->m->friendlist[i].status) {
+                /* Currently we sync all friends who we've send a friend request to.
+                   Q: do we want to be "< FRIEND_CONFIRMED?"  So we only send to
+                   confirmed friends?
+                   We don't (yet) sync the nospam, or the friend request message */
+                continue;
+            }
 
-        memset(packet, 0, sizeof(packet));
+            memset(packet, 0, sizeof(packet));
 
-        packet[0] = PACKET_ID_MDEV_SYNC;
-        packet[1] = MDEV_SYNC_CONTACT_APPEND;
+            packet[0] = PACKET_ID_MDEV_SYNC;
 
-        memcpy(packet + 2, tox->m->friendlist[i].dev_list[0].real_pk, crypto_box_PUBLICKEYBYTES);
+            if(j == 0)
```

Comment:
Code needs to match 1TBS https://en.wikipedia.org/wiki/Indent_style#Variant:_1TBS